src/Symfony/Component/Messenger/Stamp/ErrorDetailsStamp.php

```diff
@@ -32,19 +32,29 @@ final class ErrorDetailsStamp implements StampInterface
     /** @var FlattenException|null */
     private $flattenException;
 
-    public function __construct(Throwable $throwable)
+    /**
+     * @param int|mixed $exceptionCode
+     */
+    public function __construct(string $exceptionClass, $exceptionCode, string $exceptionMessage, FlattenException $flattenException = null)
```

Comment:
Isn't this a BC break when you change the constructor on a non-internal class?

Could we make sure that you still can use `new ErrorDetailsStamp($exception)`?

Reply:
@Nyholm IIRC this class is introduced in 5.2, so we should be just in time to change this.